linters/testdata/src/namedfields/namedfields.go

```diff
@@ -0,0 +1,46 @@
+package namedfields
```

Comment:
regarding test cases:
 - I think it makes some sense to have edge test cases to be sensitive for configuration change; by this I mean having a struct with the threshold number of fields and a struct with (threshold+1) number of fields
 - I would add a testcase for a struct initialization via pointer - the path that you have already covered in the linter logic